content/blog/2024-05-21-Audit.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Security Audit for Eclipse Kuksa released"
+date = "2024-05-21"
+tags = ["KUKSA.val", "Audit", "Security"]
+categories = ["Implementation"]
+banner = "img/banners/2024-05-21-auditBanner.png"
++++
+
+Applications that utilize artifacts from the Kuksa project need to rely on the secure execution of these components. We are, therefore, happy to announce the publication of an external audit report carried out by [Quarkslab](https://www.quarkslab.com). The audit was facilitated through [OSTIF](https://ostif.org) and the [Eclipse Foundation](https://www.eclipse.org) and made possible by the founding the Eclipse Foundation received from the [Alpha-Omega project](https://openssf.org/community/alpha-omega/).
+
+- The audit was scoped on the KUKSA databroker and the respective Python client SDK.
+- There were 19 findings with low or higher severity, of which two had a high severity.
+- The KUKSA team fixed most findings before the release of the report.
```

Comment:
Which ones not? And why?

Reply:
We did not fix two Info findings due to other priorities like slow handling of regex due to other priorities like reworking the API. For the level of these bullets points and even the whole blog entry I thought this would be too specific. It it still possible to read this in the full report.